### DISCUSSION TITLE
PulseTime stops countdown when wifi signal lost?

### DISCUSSION THREAD

**Discussion by jdotbdot** [arendst/Tasmota#10760] [Q&A] [answered]
As a safety precaution I use a smart plug to turn off power to my coffee machine after 30 minutes (PulseTime1 1900). 

It works fine, except if it loses wifi connection while the relay is on, the relay doesn't switch off. Presumably it stops the countdown while it searches for a wifi signal. 

Is it possible to force the relay off in the event of losing wifi?

Currently Tasmota is v8.1.0.2, flashed with Tuya-convert.

**Comment by barbudor** [comment, marked as answer]
May be a rule with `on Wifi#Disconnected do power off endon` ?
https://tasmota.github.io/docs/Rules/
Check also what is your `WifiConfig`

**Reply by ascillato** [reply]
Pulsetime don't use wifi. So, may be, you have set wificonfig to restarts your device if there is no wifi (so your pulsetime time is resetting all the time). You should use wificonfig 4 (the default)

**Reply by jdotbdot** [reply]
stat/JulesV1/RESULT = {"WifiConfig":{"2":"WifiManager"}}
Is that wrong?

Thanks for the suggestion barbudor, I'll try that rule and report back.

**Reply by ascillato** [reply]
Yes, it is wrong. Please, set wificonfig 4 and your issue is solved. No need a rule for that. Thanks.

**Comment by jdotbdot** [comment]
OK: **WifiConfig 4** results in the relay switching off after the expected time regardless of the wifi connection.
**Rule1 ON Wifi#Disconnected DO POWER OFF ENDON
Rule1 ON** results in the relay switching off immediately wifi goes down. But I didn't discover the need for **Rule1 ON** until after I'd upgraded to Tasmota 9.2.0 and set **WifiConfig 4** so not sure if it would work without wificonfig.

Thanks both for your replies!
